content/applications/websites/livechat.rst

```diff
@@ -171,27 +193,28 @@ Fill out the fields on the :guilabel:`Open: Rules` pop-up as instructed below, t
 
    .. tab:: Open automatically timer
 
```

Comment:
What about Enabled only if no operator

Reply:
Since this is only available if the Chatbot option is enabled, I didn't think it needed its own separate tab